src/app/routes/connector-ui/dashboard-page/dashboard-page/dashboard-page-data.service.ts

```diff
@@ -118,35 +116,34 @@ export class DashboardPageDataService {
   }
 
   private transferProcessKpis(): Observable<Partial<DashboardPageData>> {
-    return this.transferProcessService
-      .getAllTransferProcesses(0, 10_000_000)
+    return this.edcApiService.getTransferHistoryPage()
       .pipe(
         Fetched.wrap({
           failureMessage: 'Failed fetching transfer processes.',
         }),
         map((transferData) => ({
           incomingTransfersChart: transferData.map((it) =>
-            this.buildTransferChart(it, 'incoming'),
+            this.buildTransferChart(it.transferEntries, 'CONSUMING'),
           ),
           outgoingTransfersChart: transferData.map((it) =>
-            this.buildTransferChart(it, 'outgoing'),
+            this.buildTransferChart(it.transferEntries, 'PROVIDING'),
           ),
         })),
       );
   }
 
   private buildTransferChart(
-    transfers: TransferProcessDto[],
-    direction: 'incoming' | 'outgoing',
+    transfers: TransferHistoryEntry[],
+    direction: 'CONSUMING' | 'PROVIDING',
   ): DonutChartData {
     const filteredTransfers =
-      direction === 'incoming'
-        ? transfers.filter((it) => this.transferProcessUtils.isIncoming(it))
-        : transfers.filter((it) => this.transferProcessUtils.isOutgoing(it));
+      direction === 'CONSUMING'
+        ? transfers.filter((it) => it.direction === 'CONSUMING')
+        : transfers.filter((it) => it.direction === 'PROVIDING');
 
     // Use the keys of the TransferProcessesStates Enum as order
     const order = Object.keys(TransferProcessStates);
```

Comment:
we shouldn't rely on the order of enum values to sort the states anymore.

they are re-sorted by an order to prevent one chart having a legend "in progree, done" and the other one having one "done, in progress".

We should not rely on that enum anymore, so we should sort them by the status' status code